MVP/src/Output/categorizer.ts

```diff
@@ -42,10 +42,14 @@ export const CategorizeModules = <T>(repos: Repository<T>[]) => {
  * @param compatibleModules - An array of compatible repos. {@type Repository<T>[]}
  */
 const PrintCompatibleModules = <T>(compatibleModules: Repository<T>[]) => {
-    console.log('\nCompatible modules:');
-    compatibleModules.forEach((repo) => {
-        console.log(JSON.stringify(repo, null, 2));
-    });
+    if (compatibleModules.length === 0) {
+        console.log('\nNo compatible modules found!');
```

Comment:
Should use the Logger here so we don't have to worry about upsetting the grader

You can set the LOG_LEVEL env variable at the start of your test and still verify with the spy